src/site/content/en/handbook/inclusion-and-accessibility/index.md

```diff
@@ -1,41 +1,75 @@
 ---
 layout: handbook
 title: Inclusion and accessibility
-date: 2019-06-26
+date: 2022-02-01
 description: |
   Learn how to write for an international audience with varying needs and preferences.
 ---
 
-web.dev has a large international audience. We want members of that audience to access and understand web.dev content, and they should feel included in the ways we speak to our readers.
+web.dev has a diverse, international audience. We want all members of that
+audience to be able to access and understand web.dev content. This means our content should be both inclusive and accessible.
+
+This handbook covers some critical information for inclusion and accessibility.
+For more information, refer to the Google developer documentation style guide:
+
+*  [Writing inclusive documentation](https://developers.google.com/style/accessibility)
+*  [Writing for a global audience](https://developers.google.com/style/translation)
+*  [Writing accessible documentation](https://developers.google.com/style/accessibility).
 
 ## Use inclusive words
-Use preferred words from [our word list](https://developers.google.com/style/word-list). This list is extensive and provides both explanations and alternatives.
 
-## Use readable language
-Aim for a grade 8 reading level when possible. If you don't have a feel for what grade 8 texts look like, check out some [examples on Newsela](https://newsela.com/articles/#/rule/latest?grade_levels=8.0). You can also run your text through a readability test.
-* For pieces up to 1,600 words, use [TextEvaluator](http://textevaluator.ets.org/TextEvaluator/).
-* For pieces up to 1,000 words, use [Lexile](https://lexile.com/educators/tools-to-support-reading-at-school/tools-to-determine-a-books-complexity/the-lexile-analyzer/).
+Use preferred words from [our word
+list](https://developers.google.com/style/word-list). This list is extensive
+and provides both explanations and alternatives.
+
+### Avoid easy, simple, and fast
+
+Don't descibe tasks as easy, simple, or fast. These are often subjective bias
+based on the writer's knoweldge and resources. It's possible the reader will
+see a description of a task as "easy" and feel badly if they don't find it
+easy to complete. 
+
+You can descibe certain things as "less complex" when compared to other
+versions. But, for the most part, if you believe something is "simple," the
+example should speak for itself instead of needing extra validation.
 
-It can be difficult to write technical content for a grade 8 reading level. The following guidelines help keep your language as accessible as it can be.
+Refer to [James Fisher's blog on Don't use simply](https://jameshfisher.com/2017/02/22/dont-use-simply/)
 
-Write shorter, simpler sentences.
+## Write readable language
+
+### Aim for a grade 8 reading level
```

Comment:
This heading only makes sense to a US audience (I had to search to work out what grade 8 mapped to in the UK), and might be confusing to an international one as different countries that do have "grades" may use a completely different system. While the links below are helpful, I think a more general definition might be more inclusive as the initial heading. I've seen "School age 13/14" used in various places, granted it doesn't make for a snappy heading. In the UK we would describe that as a "Year 9 reading level".

I think the aim is to "Use straightforward English", which could then be clarified below with that being generally considered as a US Grade 8, school age 13 to 14 level of English.

Reply:
I agree with the sentiment, but for purposes of international inclusivity, I suggest we also avoid references to British educational system, as well as specific ages. For example in Serbia, they had us reading Tolstoy and the The Epic of Gilgamesh at the age of 14.

"Use straightforward English" is the general message, but the section specifically talks only about using tools to self-edit your writing, so I suggest we change the subheading to be specific about that.

Here's another tool you can add to the list, that I use most of the time: https://hemingwayapp.com/

Reply:
I wasn't suggesting we referred to the British system, the age thing is used by a number of international sources as comparable to the US-centric grade 8 idea (which appears to be in fairly standard use in the states as a shorthand for "straightforward English").

Reply:
I understood that, I was just saying that educational systems vary by country, as does the reading proficiency at certain ages. English is the official language in more countries than US and the UK and I don't think it's worthwhile thinking about how do various education systems compare and if it's fair to use any of them as a benchmark for "straightforward English".

All the tooling I came across uses the "grading" system based on US standards, there's no phrasing that can work around that, we can just acknowledge that. 

My proposal is to edit the subheading to be about "using tools to edit your writing to be more readable".